share/completions/git.fish

```diff
@@ -1426,10 +1426,22 @@ complete -f -c git -n '__fish_git_using_command reset; and not contains -- -- (c
 
 ### restore
 complete -f -c git -n '__fish_git_needs_command' -a restore -d 'Restore working tree files'
-complete -f -c git -n '__fish_git_using_command restore' -s s -l source -d 'Specify the source tree used to restore working tree'
+complete -f -c git -n '__fish_git_using_command restore' -s s -l 'source=' -d 'Specify the source tree used to restore the working tree'
 complete -f -c git -n '__fish_git_using_command restore' -s p -l patch -d 'Interactive mode'
-complete -f -c git -n '__fish_git_using_command restore' -s W -l worktree -d 'Restore working tree (default)'
+complete -f -c git -n '__fish_git_using_command restore' -s W -l worktree -d 'Restore the working tree (default)'
```

Comment:
This is unnecessary rewording - `Restore working tree` is entirely understandable, and brevity is important here because space is at a premium.